packages/css/src/tooltip/tooltip.css

```diff
@@ -0,0 +1,45 @@
+.md-tooltip {
+  z-index: 2;
+  background-color: white;
+  visibility: hidden;
+  box-shadow: 0px 4px 4px 0px #00000040;
+  border: 1px solid var(--mdPrimaryColor);
+  font-family: 'Open Sans';
+  height: fit-content;
+  width: fit-content;
+  position: absolute;
+  z-index: 99;
+  font-weight: 400;
+  font-size: 14px;
+  border-radius: 4px;
+  padding: 0.5em 1em 0.5em 1em;
+}
+
+.md-tooltip--show {
+  visibility: visible;
+}
+
+.md-tooltip--bottom {
+  transform: translate(-35%, 0.8em);
```

Comment:
Same with the different translate values. Should probably be pixels for consistency